lms/templates/courseware/info.html

```diff
@@ -70,71 +71,71 @@ <h2 class="page-subtitle">${course.display_name_with_default}</h2>
         % endif
       </div>
       <div class="info-wrapper">
-        % if user.is_authenticated():
+      % if user.is_authenticated():
         <section class="updates">
           % if studio_url is not None and masquerade and masquerade.role == 'staff':
-          <div class="wrap-instructor-info studio-view">
-            <a class="instructor-info-action" href="${studio_url}">
-              ${_("View Updates in Studio")}
-            </a>
-          </div>
+            <div class="wrap-instructor-info studio-view">
+              <a class="instructor-info-action" href="${studio_url}">
+                ${_("View Updates in Studio")}
+              </a>
+            </div>
           % endif
 
-          <h1>${_("Course Updates and News")}</h1>
+          <h4>${_("Course Updates and News")}</h4>
           ${HTML(get_course_info_section(request, masquerade_user, course, 'updates'))}
 
           ## CourseTalk widget
           % if show_coursetalk_widget:
-          <div class="coursetalk-write-reviews">
-            <div id="ct-custom-read-review-widget" data-provider="${platform_key}" data-course="${course_review_key}"></div>
-          </div>
+            <div class="coursetalk-write-reviews">
+              <div id="ct-custom-read-review-widget" data-provider="${platform_key}" data-course="${course_review_key}"></div>
+            </div>
           % endif
         </section>
+
         <section aria-label="${_('Handout Navigation')}" class="handouts">
 
-      % if SelfPacedConfiguration.current().enable_course_home_improvements:
-            <h1 class="handouts-header">${_("Important Course Dates")}</h1>
-            ## Should be organized by date, last date appearing at the bottom
-
-            % for course_date in get_course_date_blocks(course, user):
-                <div class="date-summary-container">
-                    <div class="date-summary date-summary-${course_date.css_class}">
-                        % if course_date.title:
-                            % if course_date.title == 'current_datetime':
-                                <h3 class="heading localized-datetime" data-datetime="${course_date.date}" data-string="${_(u'Today is {date}')}" data-timezone="${user_timezone}" data-language="${user_language}"></h3>
-                            % else:
-                                <h3 class="heading">${course_date.title}</h3>
```

Comment:
Some of the changes in this file, were to correct indenting and make it more readable.